revert fallback change (#3581)
plentymarkets/plugin-ceres: meta/documents/changelog_en.md

# Release Notes for plentyShop LTS

## v5.0.66 (2024-08-21) <a href="https://github.com/plentymarkets/plugin-ceres/compare/5.0.65...5.0.66" target="_blank" rel="noopener"><b>Overview of all changes</b></a>

### Changed

- The check for the payment type ‘Already paid’ has been extended.

> - A change in the LazyImg component has been undone.

Reviewer: ```suggestion
- A change in the LazyImg component was reverted.
```
